INAppStoreWindow/INAppStoreWindow.m

```diff
@@ -149,11 +149,22 @@ + (NSColor *)windowPatternOverlayColor
 
 	static dispatch_once_t oncePredicate;
 	dispatch_once(&oncePredicate, ^{
-		NSBitmapImageRep *rep = [[NSBitmapImageRep alloc] initWithData:[[NSData alloc] initWithBase64Encoding:INWindowBackgroundPatternOverlayLayer]];
+	NSBitmapImageRep *rep = nil;
+
+// initWithBase64EncodedString:options: is available in OS X v10.9 and later
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_9
+	rep = [[NSBitmapImageRep alloc] initWithData:[[NSData alloc] initWithBase64EncodedString: INWindowBackgroundPatternOverlayLayer options: NSDataBase64DecodingIgnoreUnknownCharacters]];
```

Comment:
It would be better to conditionalize the creation of the `NSData` object only, rather than repeating the `NSBitmapImageRep` creation. Same for below.